### PR TITLE
fixes selecting opponent's cards

### PR DESCRIPTION
**Pull request by andrewljohnson**

### CODE REVIEW

File: js/gamestate.js [hunk 1]
```diff
@@ -290,7 +290,7 @@ class GameState {
     }
   }
 
-  // container can be board or hand
+  // containerTypee can be board or hand
```
Comment (lacker) [inline js/gamestate.js]: spelling

File: webpack.config.js [hunk 2]
```diff
@@ -12,12 +16,15 @@ module.exports = {
...
+      { test: /\.scss$/, loader: "style!css!postcss-loader!sass" }    
 
     ]
   },
   plugins: [
     new webpack.NoErrorsPlugin()
-  ]
+  ],
+  postcss: function () {
+    return [autoprefixer, csswring]
```
Comment (lacker) [inline webpack.config.js:27]: interesting... what do these do?

Comment (andrewljohnson) [inline webpack.config.js:27]: auto-adds vendor prefixes to some css rules (-moz, -webkit, etc), after the css has been generated from scss

we're using a "transition" css rule that needs this to work in all browsers, and we will use others as time goes by

**Comment by lacker** [comment]
LGTM

**Pull request merged**